app/forms/candidate_interface/pick_course_form.rb

```diff
@@ -8,12 +8,12 @@ class PickCourseForm
     DropdownOption = Struct.new(:id, :name)
 
     def radio_available_courses
-      @radio_available_courses ||= courses_for_current_cycle.exposed_in_find.order(:name).includes(:course_options)
+      @radio_available_courses ||= courses_for_current_cycle.exposed_in_find.where('courses.applications_open_from <= ?', Time.zone.today).order(:name).includes(:course_options)
```

Comment:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/5969#pullrequestreview-794426278

Reply:
Looks like you can also extract the common query from line 67?

Reply:
Great have done 👍